### ISSUE TITLE
memleak on torchSerializeModel 

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#284]
```
==28935== 1,337 bytes in 1 blocks are definitely lost in loss record 132,250 of 132,320
==28935==    at 0x4C2FB0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==28935==    by 0x144D38: zmalloc (zmalloc.c:99)
==28935==    by 0x1C6F89: RM_Alloc (module.c:327)
==28935==    by 0x6A8B499: torchSerializeModel (torch_c.cpp:388)
==28935==    by 0x6A7A763: RAI_ModelSerializeTorch (torch.c:108)
==28935==    by 0x67E7A7A: RAI_ModelSerialize (model.c:396)
==28935==    by 0x67D823A: RedisAI_ModelGet_RedisCommand (redisai.c:717)
==28935==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28935==    by 0x13D315: call (server.c:2468)
==28935==    by 0x13DFCA: processCommand (server.c:2763)
==28935==    by 0x14F086: processInputBuffer (networking.c:1470)
==28935==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)

```

**Comment by lantiga** [comment]
Fixed in #289 (see 99b82c1)

**Comment by filipecosta90** [comment]
fixed :+1:
